manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
+PackageLocale: en-US
+Publisher: Roblox Corporation
+PackageName: Roblox

```

```suggestion
PackageName: Roblox Player
```
This should match the values written to control panel

---

apply change